bindings/pyroot/pythonizations/test/rdf_filter.py

```diff
@@ -0,0 +1,123 @@
+import unittest
+import ROOT
+import numpy as np
+
+# These imports are specific to test_with_dtypes
+import random 
+import pandas as pd
+import os
+from helper import *
+
+class PyFilter(unittest.TestCase):
+    """
+    Testing Pythonized Filters of RDF
+    """
+
+    def test_with_dtypes(self):
+        """
+        Tests the pythonized filter with all the tree datatypes and 
+        """
+        CreateData()
+        rdf = ROOT.RDataFrame("TestData", "./TestData.root")
+        test_cols =[str(c) for c in rdf.GetColumnNames()]
+        df = pd.DataFrame(rdf.AsNumpy())
+        for col_name in test_cols:
+            filter = "filter_" + TYPE_TO_SYMBOL[col_name] # string of filter name
+            func = eval(filter); # function of the filter
+            x = df[col_name].mean(); x = float(x);
+            if col_name == 'Bool_t': x = random.choice([True, False])
+            filtered = rdf.Filter(func, extra_args = {'x':x})
+            res_root = filtered.AsNumpy()[col_name]
+            if not isinstance(x, bool):
+                filtered2 = rdf.Filter(f"{col_name} > {x}")
+            else:
+                if x:
+                    filtered2 = rdf.Filter(f"{col_name} == true")
+                else:
+                    filtered2 = rdf.Filter(f"{col_name} == false")
+            res_root2 = filtered2.AsNumpy()[col_name]
+            self.assertTrue(np.array_equal(res_root,res_root2))
+
+        remaining_cols = ["Long64_t", "ULong64_t", "Long_t", "ULong_t"]
+        for col_name in remaining_cols:
+            rdf = ROOT.RDataFrame(col_name,"./TestData_" + col_name + ".root")
```

Comment:
While it is fine to have separate columns in separate files, I still find it weird that you have this custom list of remaining columns, which you test exactly like you tested other columns above. This should be done in one loop with all the columns